letsencrypt-auto-source/tests/oraclelinux6_tests.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+# Start by making sure your system is up-to-date:
+yum update -y >/dev/null
+
+LE_AUTO_PY_34="certbot/letsencrypt-auto-source/letsencrypt-auto_py_34"
+LE_AUTO="certbot/letsencrypt-auto-source/letsencrypt-auto"
+
+# Check bootstrap from current letsencrypt-auto will fail, because SCL is not enabled.
+if ! "$LE_AUTO" 2>&1 | grep -q "Enable the SCL repository and try running Certbot again."; then
```

Comment:
nit: Can we add `-n/--noninteractive/--non-interactive` here?

Reply:
Done.